lib/commonConfig.js

```diff
@@ -305,6 +307,11 @@ function processEnvironmentVariables() {
         config.autocast = process.env.IOTA_AUTOCAST === 'true';
     }
 
+    if (process.env.IOTA_MULTI_CORE) {
+        config.multiCore = process.env.IOTA_MULTI_CORE === 'true';
+    } else {
+        config.multiCore = config.multiCore === true;
```

Comment:
I understand that if no env var is provided and no "multiCore" parameter is provided in the config.js file then multicore defaults to false. That's the right behavior (to ensure backward compatibility).

(It seems so, but I'd like to double-check with an explicit confirmation :)

Reply:
Correct

Reply:
Thanks for the confirmation. NTC